app/javascript/components/users/Profile.jsx

```diff
@@ -30,6 +31,12 @@ export default function Profile() {
                       Delete Account
                     </Nav.Link>
                   </Nav.Item>
+                  <Nav.Item>
+                    <Nav.Link className="cursor-pointer" eventKey="third">
+                      <FontAwesomeIcon icon={faUser} className="pe-3" />
```

Comment:
Couldn't find a relevant icon like a lock.
I think that I'm missing something, so please fill in any suggestions.

Reply:
Right, I think we will have to swap to regular (non-svg) FontAwesome library.
SVG library is pretty limited.

Reply:
That's exactly the one that I wanted to use HH.
Can't I use it with the lib we're using now?